src/components/Header/Header.tsx

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, {
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import cn from 'classnames';
+import { TodosContext } from '../../context/TodosContext';
+import { getActiveTodos } from '../../services/todosService';
+import { addTodo, updateTodo } from '../../api/todos';
+import { USER_ID } from '../../vars/User';
+
+export const Header: React.FC = () => {
+  const {
+    todos,
+    setTodos,
+    setError,
+    setErrorTimeout,
+    setProcessingTodos,
+    setTempTodo,
+  } = useContext(TodosContext);
+  const [inputValue, setInputValue] = useState('');
+  const [isDisabled, setIsDisabled] = useState(false);
+
+  const inputField = useRef<HTMLInputElement>(null);
+
+  const toggleAllHandler = () => {
+    if (todos.find(todo => todo.completed === false)) {
+      const completedTodos = todos.filter(todo => todo.completed === false);
```

Comment:
```suggestion
    if (todos.find(todo => !todo.completed)) {
      const completedTodos = todos.filter(todo => !todo.completed);
```
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**